.travis.yml

```diff
@@ -90,16 +90,25 @@ matrix:
       - CFLAGS="$CFLAGS -I$HOME/miniconda3/include -O2 -g"
       - LDFLAGS="$LDFLAGS -L$HOME/miniconda3/lib"
 
+    - env:
```

Comment:
I added a new travis job here. If you want, I'll combine this with the `external-planarity` job.

Reply:
I would prefer it is was combined, pleased.